Xamarin.Essentials/Platform/Platform.ios.tvos.watchos.cs

```diff
@@ -73,12 +73,20 @@ internal static string GetSystemLibraryProperty(string property)
 
 #if __IOS__ || __TVOS__
 
+        static Func<UIViewController> getCurrentController;
+
+        public static void Init(Func<UIViewController> getCurrentUIViewController)
+            => getCurrentController = getCurrentUIViewController;
```

Comment:
@dimonovdd  What would be a use case for this? Just for reference.

Reply:
There was such a case in my application:
1) A popup appears
2) FilePicker appears
3) The popup closes and it closes the picker

because the picker was opened from the popup controller, and not from main navigation controller